src/hooks/useStockSelector.js

```diff
@@ -0,0 +1,15 @@
+import { useSelector } from "react-redux";
```

Comment:
Instead of making a hook
make a normal file in selector folder in store folder
Add the selector there